src/Sentry/Envelopes/Envelope.cs

```diff
@@ -47,8 +50,10 @@ public Envelope(IReadOnlyDictionary<string, object?> header, IReadOnlyList<Envel
                 ? new SentryId(guid)
                 : null;
 
-        private async Task SerializeHeaderAsync(Stream stream, IDiagnosticLogger? logger, CancellationToken cancellationToken)
+        private async Task SerializeHeaderAsync(Stream stream, IDiagnosticLogger? logger, ISystemClock clock,
+            CancellationToken cancellationToken)
```

Comment:
NIT: is this some ide setting on your machine? i am not a fan of the "wrap one parameter"

Reply:
updated